account_menu_invoice_refund/__manifest__.py

```diff
@@ -12,6 +12,5 @@
     "depends": ["account"],
     "data": ["views/account_invoice_view.xml"],
     "installable": True,
-    "development_status": "beta",
```

Comment:
see my comment

Reply:
@simahawk It still fails when I remove that commit and rebase the PR, so I guess this repository has not been updated ?
Maybe @sbidoul can help ?

Reply:
I guess we miss this https://github.com/OCA/queue/commit/eaf498011f7065076d212c401051e349d5b8db69

Reply:
@Tdu @simahawk we have the latest pylint version here. I suspect it is case sensitive. Try with `Beta`.

Reply:
@sbidoul Thanks for your help, indeed `development_status` is case sensitive